tests/conftest.py

```diff
@@ -580,6 +583,33 @@ def profiler_flags(runtime: str, profiler_type: str) -> List[str]:
     return flags
 
 
+@fixture
+def make_profiler_instance(
+    profiler_flags: List[str], monkeypatch: MonkeyPatch, output_directory: Path, profiler_state: ProfilerState
+) -> Callable[[], _GeneratorContextManager]:
+    """
+    Deliver an instance of profiler selected by runtime and profiler_type.
```

Comment:
Please leave more in-depth comment, as this method is very complex.

Reply:
👍